app-misc/task/task-2.5.1-r2.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 cmake
+
+DESCRIPTION="Taskwarrior is a command-line todo list manager"
+HOMEPAGE="https://taskwarrior.org/"
+SRC_URI="https://taskwarrior.org/download/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~x64-macos"
+IUSE="clang +sync vim-syntax zsh-completion"
```

Comment:
[Remove](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301) the vim-syntax, zsh-completion USE flags

Reply:
Should I add an elog message to tell the user that some files have been installed? I must admit I did not read those files yet.